README.md

```diff
@@ -1,25 +1,76 @@
-### Fetch secrets from AWS Secrets Manager
+# Fetch secrets from AWS Secrets Manager
 
-#### Usage
+---
 
-This small binary fetches JSON formatted secrets from AWS Secrets Manager.  
-The secrets should be in a flat JSON K:V format. 
-The secrets path should usually follow the pattern `$service_name/$env/secrets`  
+## Usage
 
+This small binary fetches JSON formatted secrets from AWS Secrets Manager.
+It exports the fetched secrets to the shell environment in which some binary (that requires said secrets) is then executed.
```

Comment:
This is slightly incorrect, it does not execute a shell of any kind, the variables are provided during the `execve` syscall, there are no other intermediary processes.